WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PersonsController < ApplicationController
+  def index
+    @regions = { 'Continent' => Continent.all.map { |continent| [continent.name, continent.id] },
+                 'Country' => Country.all.map { |country| [country.name, country.id] } }
+
+    params[:region] ||= "all"
```

Comment:
We have similar ones in a couple of places. I think it's not that bad because if it's `nil`, it actually means for us that it's `"all"` in this case. If you consider this to be really inappropriate I can use `@region` instead (we need it in the view).